client/src/style/index.css

```diff
@@ -102,6 +154,49 @@ input[type="file"] {
   display: none;
 }
 
+.greenButton {
+  background-color: #04AA6D;
+  color: white;
+  font-weight: 500;
+  min-width: 250px;
+  border: 1px solid #04AA6D;
+  border-radius: 7px;
+  margin-top: 4%;
+  display: flex;
+  justify-content: center;
+}
+
+.fieldSmall {
+  display: flex;
+  margin-bottom: 2%;
```

Comment:
hey gerard, this is looking great! one other thing I want to go over with you is the use of percentages (and to introduce a cool css tool called media queries! :)) if we have time let's tack it onto the PR review meeting

Reply:
Just changed the percentages for px in buttons and margins